crates/astria-composer/src/collectors/geth.rs

```diff
@@ -80,7 +86,12 @@ impl Status {
 
 impl Geth {
     /// Initializes a new collector instance
-    pub(crate) fn new(chain_name: String, url: String, executor_handle: executor::Handle) -> Self {
+    pub(crate) fn new(
```

Comment:
I noted that for the `grpc` server, but would like to see this here as well: can you replace this with a `GethBuilder` that has `pub(crate)` fields for each of the arguments of this constructor? These methods have a tendency to grow indefinitely and become very hard to understand.

I am fine if you do it in a followup.

Reply:
I ll do these in a future PR.